sync_first/sync_first_app/models.py

```diff
@@ -18,3 +21,6 @@ class Incident(models.Model):
     description = models.TextField(max_length=120)
     # Should by official id (tz)
     reporter = models.CharField(max_length=9, null=True, blank=True)
+
+    def __str__(self):
+        return 'Incident(main_person= '+str(self.main_person) +', spouse= '+ str(self.spouse) +', date= '+ self.date.strftime("%m/%d/%Y, %H:%M:%S") +', organization= '+self.organization+', description= '+ self.description + ', reporter= '+self.reporter + ')'
```

Comment:
nice. Notice that a date field doesn't have time, so the hours and minutes in your string are redundant